### PR TITLE
Feature/export documents api

### PR DESCRIPTION
**Pull request by tnzhu**
#### Description
Added an endpoint for exporting documents (with associated labels) for a project

#### Review Checklist
- [ ] The requirements on the description have been met
- [ ] The relevant documentation has been updated:
    - [ ] code comments for classes / methods / complex statements
    - [ ] package readme
    - [ ] documentation for high level approaches or product info
- [ ] Tests:
    - [ ] new tests written
    - [ ] existing affected tests updated
    - [ ] all green and passing
- [x] Tested by reviewer

### CODE REVIEW

**Comment by Cvos0162** [comment]
trying to test but getting 404 error

File: backend/main.py [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
 from firebase_admin import credentials
-from flask import Flask, request, make_response
+from flask import Flask
 from flask_cors import CORS
 
-from api import document_api, label_api, project_api, user_api
-
 app = Flask(__name__)
 app.register_blueprint(document_api.document_api)
 app.register_blueprint(label_api.label_api)
```
Comment (Cvos0162) [inline backend/main.py:10]: Gotta register api on the main

**Review by Cvos0162 — CHANGES_REQUESTED** [review]

File: backend/main.py (hunk 1 above)
Comment (tnzhu) [reply backend/main.py:10]: ah whoops, didn't check after merging master in. Will fix now.

**Review by Cvos0162 — APPROVED** [review]
LGTM 👍

**Pull request merged**